tests/v2/test_meetups.py

```diff
@@ -190,7 +190,7 @@ def test_no_tags_data(self):
         result = json.loads(response.data.decode('utf-8'))
         self.assertEqual(response.status_code, 400)
         self.assertEqual(result["status"], 400)
-        self.assertEqual(result["error"], 'tags field cannot be left blank')
+        self.assertEqual(result["error"], 'tags field is required')
```

Comment:
I thought the error raised was `'error':'tags field cannot be left blank'}), 400))` instead of `'tags field is required'`

Reply:
Oh! I passed the error message 'tags field is required when one passes `""` but 'tags field cannot be left blank' when one passes an empty list i.e. []. I think I should change the error message to be the same.

Reply:
If that is the case then its OK.